vendor/licenses/git_submodule/Luau.tmLanguage.dep.yml

```
@@ -0,0 +1,57 @@
+---
+name: Luau.tmLanguage
+version: bc4170802aab182127f2b6b80b0e1164f50d78c1
+type: git_submodule
+homepage: https://github.com/JohnnyMorganz/Luau.tmLanguage.git
+license: other
```

You can manually change this as licensed struggles a bit when it encounters slight variations in the licence files.

```suggestion
license: mit
```

---

> You can manually change this as licensed struggles a bit when it encounters slight variations in the license files.

I hadn't caught onto that earlier, though it's helpful, thank you! Is there anything in particular that prevents it from detecting the relevant license file?